django_celery_results/models.py

```diff
@@ -63,6 +63,10 @@ class TaskResult(models.Model):
         verbose_name=_('Result Data'),
         help_text=_('The data returned by the task.  '
                     'Use content_encoding and content_type fields to read.'))
+    date_created = models.DateTimeField(
+        auto_now_add=True, db_index=True,
+        verbose_name=_('Created DateTime'),
+        help_text=_('Datetime field when the task was created in UTC'))
```

Comment:
Is this when the task is created, or when the task result is created?

Reply:
Yes, it is. I changed the help text.

Reply:
Cool, thanks for the contribution!